bfcfg

```diff
@@ -1549,10 +1620,14 @@ arm_cfg_to_bin()
   local has_cfg_sys=0
   local has_cfg_boot=0
   local has_cfg_misc=0
-
-  if [ -n "${CTL_RESET_ALL_SYS}" ] \
-    || [ -n "${CTL_RESET_ALL_BOOT}" ] \
-    || [ -n "${CTL_RESET_ALL_MISC}" ]; then
+  local has_cfg_uefi_secureboot=0
+
+  if [ -n "${CTL_RESET_SYS}" ] \
+    || [ -n "${CTL_DELETE_ALL_BOOT}" ] \
+    || [ -n "${CTL_RESET_MISC}" ] \
+    || [ -n "${CTL_DELETE_ALL_UEFI_SECURE_BOOT_KEYS}" ] \
+    || [ -n "${CTL_UEFI_SECURE_BOOT_STATE}" ] \
```

Comment:
This is a control flag to set the UEFI Secure Boot state. It is either "ENABLED" or "DISABLED". Obviously, this parameter is effective when the platform is in user mode, i.e., PK enrolled.

Initially I was using  CTL_DISABLE_ UEFI_SECURE_BOOT, but I thought we are missing some cases where the operator switches back and forth between the enabled state and the disabled state.

Reply:
Ack. Since all the CTL_XXX configs are TRUE/FALSE thought that this would be the same. Understood now.

Reply:
I had the same line of thoughts BUT I couldn't figure out how to resolve the issue with the state. Because, users can enable it and disable it. I wanted to do something consistent with Redfish and UEFI menu.